packages/shared/tailwind.config.js

```diff
@@ -238,7 +238,21 @@ module.exports = {
     },
   },
   // eslint-disable-next-line global-require
-  plugins: [caret, typography, buttons],
+  plugins: [
+    caret, 
+    typography, 
+    buttons,
+    function ({ addUtilities }) {
+      const newUtilities = {
+        '.bg-clip-text': {
+          'background-clip': 'text',
+          '-webkit-background-clip': 'text',
+          '-webkit-text-fill-color': '#ffffff',
```

Comment:
Just to make it work with both themes 🙌 
```suggestion
          '-webkit-text-fill-color': 'var(--theme-label-primary)',
```